src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -264,7 +263,12 @@ public void Properties_AddItemToDictionary_ItemPresent()
             }
         }
 
-        [ActiveIssue(20010, TargetFrameworkMonikers.Uap)]
+        // ArgumentException: The given System.Uri cannot be converted into a Windows.Foundation.Uri.
+        // Please see http://go.microsoft.com/fwlink/?LinkID=215849 for details.
+        //
+        // This is the URI that fails the test:
+        //     LoopbackServer.GetIPv6LinkLocalAddress() == http://[fe80::3df1:c851:8775:4c5b]:25545/
```

Comment:
I've tested this and it works in both Edge and Chrome. Are you saying this is a bug in Windows::Foundation::Uri or by design?
I don't understand the link reference: isn't "http://[fe80::3df1:c851:8775:4c5b]:25545/" an absolute Uri?